.eslintrc.cjs

```diff
@@ -47,7 +47,14 @@ module.exports = defineConfig({
     'n/no-extraneous-import': [
       'error',
       {
-        allowModules: ['vite', 'less', 'sass', 'vitest', 'unbuild'],
+        allowModules: [
+          'vite',
+          'less',
+          'sass',
+          'vitest',
+          'unbuild',
+          '@polka/compression',
+        ],
```

Comment:
With this (https://github.com/vitejs/vite/pull/16146/commits/5e694e647ca9f47babf94e1e9c006ada8e605ab8) change, I guess this change is not needed anymore.

Reply:
When I remove that line, I do get this error.

<img width="780" alt="Screenshot 2024-03-11 at 23 12 22" src="https://github.com/vitejs/vite/assets/48273875/115bbec2-1242-4ee1-84cc-d03193521f68">

I think it is needed, but is there any other way out of it?

Reply:
Ah, I thought you added to `/packages/vite/package.json` instead of `/package.json`. Adding it to `/packages/vite/package.json` would work I guess.